backend/app/core/service_layer.py

```diff
@@ -28,3 +31,22 @@ def login_user_service(request: WSGIRequest, login_user_vo: LoginUserVO):
         login(request, user)
         return {'user_id': user.id}
     return User.DoesNotExist
+
+
+def serials_list_service(request: WSGIRequest, user_id: SerialsListVO):
+    try:
+        check_auth_service(request)
+        if request.user.id != user_id.user_id:
+            raise PermissionDenied
+    except PermissionDenied:
+        return HttpResponse(status=403)
+    else:
+        serials_query = list(Bookmark.objects.filter(owner=user_id.user_id).values())
+        return json.dumps(serials_query)
+
+
+def check_auth_service(request: WSGIRequest):
+    if request.user.is_authenticated:
+        return None
+    else:
+        raise PermissionDenied
```

Comment:
- use the word "validate" instead of "check". It's common practice in programming to name a function that checks something as "validate_x" in case it raises an error. If it does not raise an error (but instead returns True/False) they're called "check_x" or "verify_x".
- Use User instead of request - if you pass an object into the function just to access it's attribute - that's a signal that you need to pass just this attribute